Modules/_ssl.c

```diff
@@ -4377,7 +4377,7 @@ _ssl__SSLContext_load_dh_params_impl(PySSLContext *self, PyObject *filepath)
     FILE *f;
     DH *dh;
 
-    f = _Py_fopen_obj(filepath, "rb");
+    f = Py_fopen(filepath, "rb");
     if (f == NULL)
         return NULL;
```

Comment:
This will comply with PEP7. Now that we see it, we can help fix it, right?
```suggestion
    if (f == NULL) {
        return NULL;
    }
```
(For other changes.)

Reply:
We are trying to respect PEP 7 for new code, but my PR doesn't touch this `if (f == NULL)` in `ssl.c`. So it should be left unchanged.